PokerGame/PokerEngine/Poker.cs

```diff
@@ -7,7 +7,13 @@ public class Poker
     {
         public static List<Card> CalculateWinningHand(List<Card> handOne, List<Card> handTwo)
         {
-            List<Card> winningHand = CaclulateHandWithBestPair(handOne, handTwo);
+            List<Card> winningHand = CaclulateHandWithBestTwoPair(handOne, handTwo);
```

Comment:
use ```var```

Reply:
okay cool i think for some reason i was under the impression var didnt work for Lists but im wrong lol